test/cpp_extensions/libtorch_agnostic_2_10_extension/libtorch_agnostic_2_10/csrc/my_string_op.cpp

```diff
@@ -0,0 +1,32 @@
+#include <torch/csrc/stable/library.h>
+#include <torch/csrc/stable/device.h>
+#include <torch/csrc/stable/tensor.h>
+
+#include <string>
+#include <string_view>
+
+using torch::stable::Tensor;
+
+std::tuple<std::vector<std::string>, int64_t> my_string_op(Tensor t, std::string_view accessor) {
```

Comment:
Can you have an op that takes an `std::string` here as well?
Since we know xFormers uses that signature today, we want to make sure it does work